### PR TITLE
Nmap doc + deprecated doc automations

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Nmap README doc. Deprecated the automation scripts.

Also removed: integration-AttivoBotsink_README.md it has move to the Packs dir.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [x] Documentation (nmap readme, changelog of automation scripts + comments)

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/Legacy/Scripts/DocumentationAutomation/DocumentationAutomation.yml [hunk 1]
```diff
@@ -67,8 +67,8 @@ args:
   required: false
   secret: false
 comment: |-
-  Automates integration documentation.
-  See (https://github.com/demisto/content/tree/master/docs/integration_documentation)[https://github.com/demisto/content/tree/master/docs/integration_documentation]
+  Deprecated. The recommended way to generate documentation is via the demisto-sdk.
```
Comment (Itay4) [inline Packs/Legacy/Scripts/DocumentationAutomation/DocumentationAutomation.yml:70]: add `deprecated: true` here

File: Packs/Legacy/Scripts/HTMLDocsAutomation/HTMLDocsAutomation.yml [hunk 2]
```diff
@@ -73,8 +73,8 @@ args:
   required: false
   secret: false
 comment: |-
-  Automates integration documentation.
-  See (https://github.com/demisto/content/tree/master/docs/integration_documentation)[https://github.com/demisto/content/tree/master/docs/integration_documentation]
+  Deprecated. The recommended way to generate documentation is via the demisto-sdk.
```
Comment (Itay4) [inline Packs/Legacy/Scripts/HTMLDocsAutomation/HTMLDocsAutomation.yml:76]: add `deprecated: true` here

File: Packs/Nmap/Integrations/Nmap/README.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
+Run nmap scans with the given parameters.
+This integration was integrated and tested with version 7.70 of nmap. The nmap binary is shipped with the Integration docker. You can see the options available for running an nmap scan here: https://nmap.org/book/man-briefoptions.html.
```
Comment (Itay4) [inline Packs/Nmap/Integrations/Nmap/README.md:2]:
```suggestion
This integration was integrated and tested with version 7.70 of nmap. The nmap binary is shipped with the integration docker. You can see the options available for running an nmap scan here: https://nmap.org/book/man-briefoptions.html.
```

File: Packs/Legacy/Scripts/HTMLDocsAutomation/HTMLDocsAutomation.yml (hunk 2 above)
Comment (glicht) [reply Packs/Legacy/Scripts/HTMLDocsAutomation/HTMLDocsAutomation.yml:76]: done

File: Packs/Legacy/Scripts/DocumentationAutomation/DocumentationAutomation.yml (hunk 1 above)
Comment (glicht) [reply Packs/Legacy/Scripts/DocumentationAutomation/DocumentationAutomation.yml:70]: done

**Comment by glicht** [comment]
@kirbles19 waiting for your review

**Comment by kirbles19** [comment]
@glicht Done.

**Pull request merged**